### PR TITLE
Release workflow for uploading apps and updating the version

### PR DESCRIPTION
**Pull request by sebastian-quintero**
# Description

This PR creates the release workflow for uploading apps and updating their version. The `release.yml` file was created and merged because I needed the file to be present to be able to test workflows. Once the apps are uploaded with the new versions and the manifest is updated, the bot commits the new version files to the repo.

### CODE REVIEW

File: knapsack-gosdk/VERSION [hunk 1]
```diff
@@ -1 +1 @@
-v1.2.2
+v1.2.0
```
Comment (sebastian-quintero) [inline knapsack-gosdk/VERSION:1]: This was part of testing. I'm putting things back to normal.

File: knapsack-java-ortools/VERSION [hunk 2]
```diff
@@ -1 +1 @@
-v1.2.2
+v1.2.0
```
Comment (sebastian-quintero) [inline knapsack-java-ortools/VERSION:1]: Part of testing.

**Review by cowanwalls — APPROVED** [review]

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -35,19 +36,41 @@ jobs:
...
+          mkdir -p ~/.ssh
+          ssh-keyscan github.com >> ~/.ssh/known_hosts
+          ssh-agent -a $SSH_AUTH_SOCK > /dev/null
+          ssh-add - <<< "${{ secrets.NEXTMVBOT_SSH_KEY }}"
+
+          echo "${{ secrets.NEXTMVBOT_SIGNING_KEY }}" > ~/.ssh/signing.key
+          chmod 600 ~/.ssh/signing.key
+
+          git config --global user.name "nextmv-bot"
+          git config --global user.email "tech+gh-nextmv-bot@nextmv.io"
```
Comment (nilsbeck) [inline .github/workflows/release.yml:56]: Is this good practice to use this kind of a mail in a public repository? this looks like it is showing internals?
maybe I am just paranoid.

**Review by nilsbeck — COMMENTED** [review]
removed

**Pull request merged**